### PR TITLE
fix: Minor UI Changes

### PR DESCRIPTION
**Pull request by techwithanirudh**
﻿Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation update  
  

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
##

##

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: client/src/components/Nav/NavLinks.jsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { cn } from '~/utils/';
 import DotsIcon from '../svg/DotsIcon';
 
 import store from '~/store';
+import LinkIcon from '../svg/LinkIcon';
```
Comment (danny-avila) [inline client/src/components/Nav/NavLinks.jsx]: Add LinkIcon to svg/index and import from ~/components

File: client/src/components/svg/LinkIcon.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+export default function LinkIcon() {
```
Comment (danny-avila) [inline client/src/components/svg/LinkIcon.jsx]: Change this file to tsx

File: client/src/components/Nav/NavLinks.jsx [hunk 3]
```diff
@@ -15,6 +15,7 @@ import { cn } from '~/utils/';
 import DotsIcon from '../svg/DotsIcon';
```
Comment (danny-avila) [inline client/src/components/Nav/NavLinks.jsx]: Please make sure all SVGs are exported from svg/index.ts

File: client/src/components/Nav/NavLinks.jsx [hunk 4]
```diff
@@ -15,6 +15,7 @@ import { cn } from '~/utils/';
...
 
 import store from '~/store';
+import { LinkIcon } from '~/components/svg';
```
Comment (danny-avila) [inline client/src/components/Nav/NavLinks.jsx]: Also, import from `~/components` not `~/components/svg`

**Comment by techwithanirudh** [comment]
@danny-avila Fixed!

File: client/src/components/Nav/NavLinks.jsx [hunk 5]
```diff
@@ -12,9 +12,9 @@ import ClearConvos from './ClearConvos';
 import Logout from './Logout';
```
Comment (danny-avila) [inline client/src/components/Nav/NavLinks.jsx:10]: you missed TrashIcon, and GearIcon

File: client/src/components/Nav/index.jsx [hunk 6]
```diff
@@ -223,7 +223,7 @@ export default function Nav({ navVisible, setNavVisible }) {
       {!navVisible && (
         <button
           type="button"
-          className="nav-open-button mt-1 fixed left-2 top-0.5 z-10 inline-flex h-11 w-11 items-center justify-center rounded-md border border-black/10 dark:border-white/20 bg-white dark:bg-gray-800 text-black hover:text-gray-500 dark:text-white dark:hover:text-gray-400"
+          className="nav-open-button mt-1 fixed left-2 top-0.5 z-10 flex p-3 items-center gap-3 transition-colors duration-200 text-white cursor-pointer text-sm rounded-md border bg-white dark:bg-gray-800 border-black/10 dark:border-white/20 hover:bg-gray-50 dark:hover:bg-gray-700 h-11 w-11"
```
Comment (danny-avila) [inline client/src/components/Nav/index.jsx]: This makes the nav button invisible on light mode 

image

**Review by danny-avila — CHANGES_REQUESTED** [review]
Please see my remaining comments

**Comment by techwithanirudh** [comment]
@danny-avila Fixed!

**Review by danny-avila — APPROVED** [review]

**Pull request merged**